pyproject.toml

```diff
@@ -44,6 +44,7 @@ classifiers = [
     'Programming Language :: Python :: 3.11',
     'Programming Language :: Python :: 3.12',
     'Programming Language :: Python :: 3.13',
+    'Programming Language :: Python :: 3.14',
```

Comment:
I think this classifier is needed on the main branch as well

Reply:
This PR is targetting main, but with the 2.3.3 milestone so it gets backported afterwards?